src/common/src/session_config/mod.rs

```diff
@@ -50,12 +52,14 @@ pub enum SessionConfigError {
 
 type SessionConfigResult<T> = std::result::Result<T, SessionConfigError>;
 
+#[serde_as]
 /// This is the Session Config of RisingWave.
-#[derive(SessionConfig)]
-pub struct ConfigMap {
+#[derive(Clone, Debug, Deserialize, Serialize, SessionConfig, ConfigDoc, PartialEq)]
+pub struct SessionConfig {
     /// If `RW_IMPLICIT_FLUSH` is on, then every INSERT/UPDATE/DELETE statement will block
     /// until the entire dataflow is refreshed. In other words, every related table & MV will
     /// be able to see the write.
+    #[serde(rename = "rw_implicit_flush")]
     #[parameter(default = false, rename = "rw_implicit_flush")]
     implicit_flush: bool,
```

Comment:
2 renames here 😄. How about just renaming the field to `rw_implicit_flush`

Reply:
Rename this will cause too many unrelated changes to this pr. Maybe later.